Makefile

```diff
@@ -13,14 +13,14 @@ install-test-rode:
 install-test-rode-ui:
 	ct install --target-branch main --charts charts/rode-ui --helm-extra-args --timeout=10m
 
-install-test-collectors-setup:
+install-test-collectors-enforcers-setup:
```

Comment:
Open to alternative names for these targets, but since it's not just collectors now I felt obligated to change the name